### PR TITLE
fix : Meta tags input in help center clears the input value after clicking outside

### PR DESCRIPTION
**Pull request by gitstart**
Fixes https://github.com/chatwoot/chatwoot/issues/5482

## Description

When we enter something in the meta tags input of the article and then clicked outside, the entered value of meta tags input gets cleared. This PR implements the preserving the search value in meta tags input.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Go to an article
- Enter something in meta tags input and click outside
- When we again go to meta tags input, the previously entered input is preserved

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Demo video
https://www.loom.com/share/09080e29cea1438baf5ed8a17363cb61

### CODE REVIEW

**Comment by iamsivin** [comment]
Issue https://github.com/chatwoot/chatwoot/issues/5482

File: app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue [hunk 1]
```diff
@@ -87,6 +87,7 @@
             track-by="name"
             :multiple="true"
             :taggable="true"
+            :preserve-search="true"
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue:90]: What does the `preserve-search` do here? Doesn't seem like a valid prop for the element? As far as I know, there is no standard HTML attribute or property named `preserve-search` either

**Review by scmmishra — COMMENTED** [review]
About the behaviour of this, given there is no explicit "save" button. On blur, we should save what the user has entered as tags. 

WDYT @pranavrajs @nithindavid?

**Comment by pranavrajs** [comment]
> About the behaviour of this, given there is no explicit "save" button. On blur, we should save what the user has entered as tags.

Yes, we should save the tags on blur.

File: app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue (hunk 1 above)
Comment (scmmishra) [reply app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue]: Noted

File: app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue [hunk 2]
```diff
@@ -184,13 +186,18 @@ export default {
...
-      this.metaTags.push(tag);
-      this.$refs.tagInput.$el.focus();
+      const tags = tagValue.split(',').map(value => ({
+        name: value.trim(),
+      }));
+      this.metaTags.push(...tags);
       this.saveArticle();
     },
+    onBlur() {
+      const currentSearch = this.$refs.tagInput.$refs.search.value;
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue:196]: This looks a bit hacky, but the `vue-multiselect` is not the ideal candidate for this kind of component with arbitrary tags and no list to select form. 

An alternative approach is that we could listen to the `@search-change` event. And store the result in a variable, onBlur we just assign the value to `currentSearch` and then reset the variable.

This is cleaner and does not leak the implementation details of the underlying component

**Review by scmmishra — CHANGES_REQUESTED** [review]

File: app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue (hunk 2 above)
Comment (gitstart) [reply app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue]: @scmmishra Thanks for the suggestion, it works fine.

**Review by scmmishra — APPROVED** [review]

**Pull request merged**